### PR TITLE
Issue #11 ruby gem

### PR DESCRIPTION
**Pull request by flbulgarelli**
There are some tasks pending than only owner can do:

- [x] Activate travis
- [x] Check if the gem email is fine 
- [x] Setup travis encrypted deployment keys for rubygems
- [x] Do the first manual push of the gem

### CODE REVIEW

**Comment by flbulgarelli** [comment]
:warning:  I had to create a `dist` task, but it is not fully functional since it is not actually generating the `gs-board.html`, probably because I am not executing the `symlinks` and `vulcanize`  tasks. 

I haven't add them to dist since they look somewhat coupled to the `demo` task, and a refactor my be necessary. However, I don 't still get some aspects of the build process so I think I need @rodri042 assistance here :smile:

**Comment by luchotc** [comment]
If possible, it would be nice to extract the images in "htmls/components" to their own path

File: gem/Rakefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+RSpec::Core::RakeTask.new(:spec)
+
+task :default => :spec
+
+namespace :wrapper do
+  desc 'Wrap styles assets into this gem'
+  task wrap: :clean do
+    #  gs-board
+    cp_r '../dist/gs-board.html', 'app/assets/htmls'
+    cp_r '../dist/components', 'app/assets/htmls'
```
Comment (rodri042) [inline gem/Rakefile]: This isn't needed since `vulcanize` creates the `gs-board.html` file with all the components bundled

**Comment by afska** [comment]
I'm gonna merge this and push the changes to generate the `gs-board.html` file

**Pull request merged**